src/deprecated/routes/RequestsRoute/RequestsRoute.js

```diff
@@ -241,13 +254,13 @@ export const getListFormatter = (
     />),
   'itemBarcode': rq => (rq?.item?.barcode || <NoValue />),
   'position': rq => (rq.position || <NoValue />),
-  'proxy': rq => (rq.proxy ? getFullName(rq.proxy) : <NoValue />),
+  'proxy': rq => userFormatter(rq.proxyUserId, rq.proxy),
```

Comment:
From https://folio-org.atlassian.net/browse/UXPROD-4302
`And, if they exist, the proxy.lastName, proxy.firstName, proxy.barcode, and proxyUserId are removed from the request record`
In story we do not have clear scenarios and difficult to understand correct behavior for current case.
If proxyUserId will be removed (not change to null) we do not need and we cannot to use userFormatter here.
Could you please double check can/need we show anonymized for proxy?